### DISCUSSION TITLE
Can you use Tailscale SSH with your addon by advertising using "tailscale up --ssh" or similar?

### DISCUSSION THREAD

**Discussion by RobWills** [hassio-addons/app-tailscale#374] [Ideas]
Firstly using your tailscale addon has massively helped with my remote access to my home assistant server for which I thank you.

Would tailscale ssh work out of the box if using your add-on installed on HAOS or would this require additional development? I've used it to access AWS linux servers and it is so useful not having to setup the private and public keys and being able to access from anywhere.

I was going to try in the Terminal addon using:

tailscale up --ssh

However, my linux knowledge is embarrassing these days and I couldn't even find out where tailscale was installed on my server.

**Comment by frenck** [comment]
Sorry, that is not possible.

It would only bring you into an isolated container, not Home Assistant.

../Frenck

**Reply by RobWills** [reply]
Ah OK, that's a shame. Thank you.
